integration-test/integcb/Profile_template

```diff
@@ -19,12 +19,14 @@ export CLUSTERPROXY_ENABLED=true
 export ENVIRONMENT_AUTOSYNC_ENABLED=false
 export ENVIRONMENT_FREEIPA_SYNCHRONIZEONSTART=false
 export ENVIRONMENT_EXPERIENCE_SCAN_ENABLED=true
-export CB_LOCAL_DEV_LIST=jaeger,environments2-api,audit-api,distrox-api,thunderhead-api,datalake-api
+export DATALAKE_DR_ENABLED=true
+export CB_LOCAL_DEV_LIST=jaeger,environments2-api,audit-api,distrox-api,thunderhead-api,datalake-api,datalake-dr
 
 export ALTUS_AUDIT_ENDPOINT=thunderhead-mock
 export INTEGRATIONTEST_AUTHDISTRIBUTOR_HOST=thunderhead-mock
 export AUTHDISTRIBUTOR_HOST=thunderhead-mock
 export CLUSTERDNS_HOST=thunderhead-mock
+export DATALAKE_DR_ENDPOINT=thunderhead-mock:8981
 export MOCK_INFRASTRUCTURE_HOST=mock-infrastructure
 
 export CPUS_FOR_CLOUDBREAK=8.0
```

Comment:
Is this file used only for mock tests?

Reply:
Yes this is used only for the tests run in the PR which all use the mock cloud provider.